### ISSUE TITLE
Hunter should build new tents

### ISSUE THREAD

**Issue by maxanier** [TeamLapen/Vampirism#118]
and maybe trade with players

**Comment by tech-boy101** [comment]
if or go be do this way i say that you see so many tents so way to stop this form happen to is to make a thing that vampire hunter spawn the world nomal but rare and set besa the older the hunter place is the bigger it will be so go to small camp to a gaint strong hold if dont understand what my say piz tell me

**Comment by maxanier** [comment]
Yeah, when I implement this, I will lower the spawn rate for tents. Some lager camps sounds cool too

**Comment by EpicCreeperHD** [comment]
This is something people on my server have been doing. Building big anti-vampire camps around the place over tents and stuff so that they have the NPC hunters defending it while they're away.

They suggested an idea where they could build their own camp and if they placed a tent and then placed an item or object into the tent it would spawn hunters but like villagers (if there's too little in the area, spawn more).

And I'd love to see a Fort Dawnguard style of hunter camp. Y'know the big castle outfitted with hunter equipment and the smaller camps around it?

**Comment by Cheaterpaul** [comment]
This is something i have not seen in other mods. Seems like a rather complicated ai feature.

**Comment by maxanier** [comment]
But I don't think "not seen in other mods" should be a criteria. Would be pretty boring if Vampirism just does the same thing as other mods do ;)

But yes, will have many implications and need extensive testing
